cmake/Modules/Testing.cmake

```diff
@@ -4,48 +4,7 @@
 option(ENABLE_TESTING "Enable testing" OFF)
 if(ENABLE_TESTING)
   enable_testing()
-  option(LAMMPS_TESTING_SOURCE_DIR "Location of lammps-testing source directory" "")
-  option(LAMMPS_TESTING_GIT_TAG    "Git tag of lammps-testing" "master")
-  mark_as_advanced(LAMMPS_TESTING_SOURCE_DIR LAMMPS_TESTING_GIT_TAG)
-
-  if (CMAKE_VERSION VERSION_GREATER "3.10.3" AND NOT LAMMPS_TESTING_SOURCE_DIR)
-    include(FetchContent)
-
-    FetchContent_Declare(lammps-testing
-      GIT_REPOSITORY https://github.com/lammps/lammps-testing.git
-      GIT_TAG ${LAMMPS_TESTING_GIT_TAG}
-    )
-
-    FetchContent_GetProperties(lammps-testing)
-    if(NOT lammps-testing_POPULATED)
-      message(STATUS "Downloading tests...")
-      FetchContent_Populate(lammps-testing)
-    endif()
-
-    set(LAMMPS_TESTING_SOURCE_DIR ${lammps-testing_SOURCE_DIR})
-  elseif(NOT LAMMPS_TESTING_SOURCE_DIR)
-    message(WARNING "Full test-suite requires CMake >= 3.11 or copy of\n"
-                    "https://github.com/lammps/lammps-testing in LAMMPS_TESTING_SOURCE_DIR")
-  endif()
-
-  add_test(NAME ShowHelp COMMAND $<TARGET_FILE:lmp> -help)
```

Comment:
I think, trying to run `lmp` itself has been proven very useful in the past to check if the executable is actually runable.

Reply:
being able to run a test executable is providing the same level of information and more.

Reply:
Are the test excutable linked exactly with the same flags, libraries etc?

Reply:
One issue I had in the past was something with NetCDF, where the compile worked, but when running lammps it fail an error message about an incompatible NetCDF. That case would be cover by linking the lammps library into a test executable I guess.